packages/babel-plugin-emotion/src/index.js

```diff
@@ -441,6 +442,8 @@ export default function(babel: Babel) {
             // path.hub.file.opts.filename !== 'unknown' ||
             state.opts.extractStatic
 
+          state.outputDir = state.opts.outputDir
```

Comment:
There's no need to do this, just use `state.opts.outputDir`. The only reason it's done for `extractStatic` is because it used to be determined by other things like as you can see where there was a filename passed to babel.